packages/core/src/components/TextArea/constants.ts

```diff
@@ -0,0 +1,4 @@
+// eslint-disable-next-line import/prefer-default-export
+export const LANGUAGE_RULES = {
+  UNNECESSARY_ADMIN_NOTE_RULE: 'AGENT_TOOLS_UNNECESSARY_ADMIN_NOTE_RULE',
```

Comment:
This is tying third-party code to an open source project, so will not be accepted. I think the best solution would be to add new props to proofreader that check for secondary/highlighted state, for example.

```ts
isRuleHighlighted?: (rule: ProofreadRuleMatch) => boolean;
isRuleSecondary?: (rule: ProofreadRuleMatch) => boolean;
```

This is a similar implmentation to props on `Autocomplete`.